domains/transfer/Transfer.API/API/Controllers/CvrController.cs

```diff
@@ -0,0 +1,104 @@
+using System.Linq;
+using System.Threading.Tasks;
+using API.ApiModels.Responses;
+using API.Clients.Cvr;
+using API.Models;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace API.Controllers;
+
+[Authorize]
+[ApiController]
+[Route("api/cvr")]
+public class CvrController : Controller
+{
+    private readonly CvrClient client;
+
+    public CvrController(CvrClient client)
+    {
+        this.client = client;
+    }
+
+    /// <summary>
+    /// Get CVR registered company information
+    /// </summary>
+    /// <param name="cvrNumber">CVR number of the company</param>
+    /// <response code="200">Successful operation</response>
+    /// <response code="404">CVR company not found</response>
+    [ProducesResponseType(typeof(CvrCompanyDto), 200)]
+    [ProducesResponseType(typeof(void), 404)]
+    [HttpGet("{cvrNumber}")]
+    public async Task<ActionResult<CvrCompanyDto>> GetCvrCompany(string cvrNumber)
+    {
+        var cvr = CvrNumber.TryParse(cvrNumber);
+        if (cvr == null)
+            return NotFound();
```

Comment:
The other controllers in this project uses FluentValidation for this case

Reply:
How do you make one for just a string?

Reply:
stays the same